docker-storage-setup.sh

```diff
@@ -419,35 +419,32 @@ systemd_escaped_filename () {
   echo $filename
 }
 
-reset_docker_root_volume () {
+reset_extra_volume () {
   local mp filename
-  if docker_root_volume_exists; then
-    mp=$(docker_root_lv_mountpoint)
+  if extra_volume_exists $1; then
+    mp=$(extra_lv_mountpoint $1)
     if [ -n "$mp" ];then
       if ! umount $mp >/dev/null 2>&1; then
         Fatal "Failed to unmount $mp"
       fi
     fi
-    lvchange -an $VG/${DOCKER_ROOT_LV_NAME}
-    lvremove $VG/${DOCKER_ROOT_LV_NAME}
+    lvchange -an $VG/${1}
+    lvremove $VG/${1}
```

Comment:
Instead of using $1 and $2, can we save these parameters in variable names in the beginning of funciton. That makes it little easy to read.

lv_name=$1
mount_dir=$2

And then use `$lv_name` and `$mount_dir` in rest of the code.